cli/src/commands/uninstall.rs

```diff
@@ -1,53 +1,51 @@
 //! Phylum CLI removal.
 
+use std::fs;
 use std::io::Write;
 use std::path::{Path, PathBuf};
-use std::{env, fs};
 
 use anyhow::{anyhow, Result};
 use clap::ArgMatches;
 use tempfile::NamedTempFile;
 
 use crate::commands::{CommandResult, ExitCode};
-use crate::{config, print_user_success, print_user_warning};
+use crate::{dirs, print_user_success, print_user_warning};
 
 /// Handle the `uninstall` subcommand.
 pub fn handle_uninstall(matches: &ArgMatches) -> CommandResult {
-    let home_dir = home_dir()?;
-
     if matches.is_present("purge") {
-        purge(&home_dir);
+        purge()?;
     }
 
-    remove_installed_files(&home_dir);
+    remove_installed_files()?;
 
-    if let Err(err) = Shell::Bash.cleanup(&home_dir) {
+    if let Err(err) = Shell::Bash.cleanup() {
         print_user_warning!("BASH config error: {}", err);
     }
 
-    if let Err(err) = Shell::Zsh.cleanup(&home_dir) {
+    if let Err(err) = Shell::Zsh.cleanup() {
         print_user_warning!("ZSH config error: {}", err);
     }
 
     Ok(ExitCode::Ok.into())
 }
 
 /// Remove the entire `~/.config/phylum` directory.
```

Comment:
Not due to the changes in this review...just noticed now...but this comment is too specific. Instead, it could be:

`/// Remove the entire phylum config directory.`